### PR TITLE
Updated Parts

### PR DESCRIPTION
**Pull request by jalogisch**
I have updated the Getting Started section and added some links additional links to the documentation and some more text.

addiditional i removed the not working streaming dashboard

### CODE REVIEW

File: pages/getting_started.rst [hunk 1]
```diff
@@ -4,7 +4,13 @@
 Getting Started
 ***************
 
-The fastest way to evaluate Graylog is to download the virtual appliance file and run it in VMWare or VirtualBox.
+To get the first look what Graylog is and how you can work with we have a :ref:`virtual appliance <virtual-machine-appliances>` to run on VMWare or Virtualbox. The following will guide you to start using Graylog.
```
Comment (mariussturm) [inline pages/getting_started.rst]: * a first look
  * work with it
  * that runs on
  * guide you through the first steps of using Graylog

File: pages/getting_started.rst [hunk 2]
```diff
@@ -4,7 +4,13 @@
...
+
+After you feel omfortable you should make a fresh :ref:`installation <installing>` that takes care of youre recommendation regarding the sizing and the security settings. The virtual appliance is only created to have a showcase about the features and is not created to run in production.
```
Comment (mariussturm) [inline pages/getting_started.rst]: * comfortable
  * create a fresh
  * that follows your own requirements

File: pages/getting_started.rst [hunk 3]
```diff
@@ -4,7 +4,13 @@
...
+
+If you need assistance to plan and build your logging environment `we offer professional support <https://www.graylog.org/professional-support>`__ that can work with you.
```
Comment (mariussturm) [inline pages/getting_started.rst]: * assistance in planning or building
  * -(that can work with you)

File: pages/getting_started/check_messages.rst [hunk 4]
```diff
@@ -27,3 +27,4 @@ If You Don't Have Messages
 
 ``$ sudo netstat -peanut | grep ":5140"``
 
+If you still have issues, connect to our `community support <https://www.graylog.org/community-support>`__ or get in touch with us about `professional support <https://www.graylog.org/professional-support>`__.
```
Comment (mariussturm) [inline pages/getting_started/check_messages.rst]: * about -> via the professional support offering

File: pages/getting_started/rsyslog.rst [hunk 5]
```diff
@@ -30,3 +30,5 @@ Type::
   $sudo service rsyslog restart
 
 If you have modified the config file and it is somehow invalid, the service command will not bring rsyslog back up - so don't worry, you can always delete the line!
+
+We have even more information how to send messages to Graylog on :ref:`a special page <ingest_data>` in our Documentation.
```
Comment (mariussturm) [inline pages/getting_started/rsyslog.rst]: Further informations on how to send log messages are on a separate page in our documentation

**Comment by mariussturm** [comment]
LGTM!

File: pages/getting_started.rst [hunk 6]
```diff
@@ -4,11 +4,11 @@
 Getting Started
 ***************
 
-To get the first look what Graylog is and how you can work with we have a :ref:`virtual appliance <virtual-machine-appliances>` to run on VMWare or Virtualbox. The following will guide you to start using Graylog.
+To get a first look what Graylog is and how you can work with it we have a :ref:`virtual appliance <virtual-machine-appliances>` that runs on VMWare or Virtualbox. The following will guide you through the fist steps of using Graylog.
```
Comment (mariussturm) [inline pages/getting_started.rst:7]: fist -> first

Comment (jalogisch) [reply pages/getting_started.rst:7]: cherry picked into master!

**Pull request merged**